deploy/runtools/runtime_config.py

```diff
@@ -316,7 +316,7 @@ def __init__(self, runtimeconfigfile: str, configoverridedata: str) -> None:
         self.print_cycle_prefix = True
 
         if 'tracing' in runtime_dict:
-            self.trace_enable = runtime_dict['tracing'].get('enable') == "yes"
+            self.trace_enable = runtime_dict['tracing'].get('enable', False)
```

Comment:
So what happens if yes or no is misspelled? Does `get` have to return the same type as the default?

Reply:
No it would return a string. I assumed that things would have to be spelled correctly.

Reply:
Can you enforce that it returns a boolean type in this PR.

Reply:
This is an area where having a schema for the config files would be helpful.  Also note that if we're going to depend on PyYAML's implicit converson of unquoted yes/no, we're depending on YAML 1.1 and should probably document that. see https://stackoverflow.com/questions/36463531/pyyaml-automatically-converting-certain-keys-to-boolean-values

Reply:
I too think we should have a schema for all of these files.

Reply:
I'll make this TODO #123122232. But I'll also enforce that it returns a bool